application/ui/widgets/eh-widgets-dir/eh-widgets/src/helpers/validation/bundleGroupSchema.js

```diff
@@ -16,7 +16,7 @@ export const bundleGroupSchema = Yup.object().shape({
     )
     .required("Documentation is a required field"),
   status: Yup.string().required("Status is a required field"),
-  version: Yup.string().matches(/^([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?$/gm, "Version must match semantic versioning format (e.g. x.x.x)").required("Version is a required field"),
+  version: Yup.string().matches(/^v([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?$/gm, "Version must match semantic versioning format (e.g. vx.x.x)").required("Version is a required field"),
```

Comment:
v is optional so the comment should indicate either vx.x.x or x.x.x

Reply:
ok, fix it asap

Reply:
done!